### ISSUE TITLE
 Golden can't save my workouts

### ISSUE THREAD

**Issue by gomeskiller** [GoldenCheetah/GoldenCheetah#4184]
Issue tracker is **only** for Bugs and Features, please don't open issues for questions or technical support. Before to open a new issue please read the contributing guidelines (link below).

If you have questions, please read the FAQs and User's/Developer's Guide:
* FAQs - https://github.com/GoldenCheetah/GoldenCheetah/wiki/FAQ
* User's Guide - https://github.com/GoldenCheetah/GoldenCheetah/wiki/UG_Main-Page_Users-Guide
* Developer's Guide - https://github.com/GoldenCheetah/GoldenCheetah/wiki/Developers-guide

If you need help or technical support please use the forums:
* Users - https://groups.google.com/forum/#!forum/golden-cheetah-users
* Developers - https://groups.google.com/forum/#!forum/golden-cheetah-developers

**Comment by gomeskiller** [comment]
![Golden](https://user-images.githubusercontent.com/97481669/156156365-43542053-bdb3-4381-9db9-74ab61050865.PNG)

**Comment by gomeskiller** [comment]
If you can help me, it will be from the program or from the pc
